index.bs

```diff
@@ -1960,22 +1981,14 @@ Instances of <code>ReadableByteStream</code> are created with the internal slots
     <td>A value indicating how the stream failed, to be given as a failure reason or exception when trying to operate
       on an errored stream
   </tr>
-  <tr>
-    <td>\[[underlyingByteSource]]
```

Comment:
I think this change, and the accompanying ones where we extract functions from the UBS, is not great. We should keep a reference to it and call its methods as methods. See https://github.com/whatwg/streams/issues/256 and https://github.com/whatwg/streams/pull/265.